Remove these messages:
Text.pm: Use of qw(...) as parentheses is deprecated at /usr/share/BackupPC/lib/BackupPC/Storage/Text.pm line 301.
Lib.pm: Use of qw(...) as parentheses is deprecated at /usr/share/BackupPC/lib/BackupPC/Lib.pm line 1412.

diff -Naur BackupPC-3.2.1-pristine/lib/BackupPC/Lib.pm BackupPC-3.2.1/lib/BackupPC/Lib.pm
--- BackupPC-3.2.1-pristine/lib/BackupPC/Lib.pm	2012-12-24 15:01:44.518877629 -0700
+++ BackupPC-3.2.1/lib/BackupPC/Lib.pm	2011-04-24 21:31:55.000000000 -0600
@@ -1409,7 +1409,7 @@
 
     $conf->{$shareName} = [ $conf->{$shareName} ]
                     if ( ref($conf->{$shareName}) ne "ARRAY" );
-    foreach my $param qw(BackupFilesOnly BackupFilesExclude) {
+    foreach my $param (qw(BackupFilesOnly BackupFilesExclude)) {
         next if ( !defined($conf->{$param}) );
         if ( ref($conf->{$param}) eq "HASH" ) {
             #
diff -Naur BackupPC-3.2.1-pristine/lib/BackupPC/Storage/Text.pm BackupPC-3.2.1/lib/BackupPC/Storage/Text.pm
--- BackupPC-3.2.1-pristine/lib/BackupPC/Storage/Text.pm	2012-12-24 15:03:16.017830715 -0700
+++ BackupPC-3.2.1/lib/BackupPC/Storage/Text.pm	2011-04-24 21:31:55.000000000 -0600
@@ -298,7 +298,7 @@
     #
     # Promote BackupFilesOnly and BackupFilesExclude to hashes
     #
-    foreach my $param qw(BackupFilesOnly BackupFilesExclude) {
+    foreach my $param (qw(BackupFilesOnly BackupFilesExclude)) {
         next if ( !defined($conf->{$param}) || ref($conf->{$param}) eq "HASH" );
         $conf->{$param} = [ $conf->{$param} ]
                                 if ( ref($conf->{$param}) ne "ARRAY" );
